wolox-books/src/views/BookDetail.vue

```diff
@@ -0,0 +1,15 @@
+<template lang='pug'>
+div
+  navbar
```

Comment:
Aren't you using this `navbar` in every component? Try moving it to another higher component and add some logic to it :smile: